FirebaseAuth/Sources/Backend/RPC/FIRVerifyAssertionRequest.m

```diff
@@ -84,10 +84,27 @@
  */
 static NSString *const kReturnSecureTokenKey = @"returnSecureToken";
 
-/** @var kDisplayName
-    @brief The key for the "displayName" value in the request.
+/** @var kUserKey
+    @brief The key for the "user" value in the request. The value is a JSON object that contains the
+   name of the user.
  */
-static NSString *const kDisplayNameKey = @"displayName";
+static NSString *const kUserKey = @"user";
+
+/** @var kNameKey
+    @brief The key for the "name" value in the request. The value is a JSON object that contains the
+   first and/or last name of the user.
+ */
+static NSString *const kNameKey = @"name";
+
+/** @var kFirstNameKey
+    @brief The key for the "firstName" value in the request.
+ */
+static NSString *const kFirstNameKey = @"firstName";
+
+/** @var kLastNameKey
+    @brief The key for the "lastName" value in the request.
+ */
+static NSString *const kLastNameKey = @"lastName";
 
 /** @var kReturnIDPCredentialKey
```

Comment:
The variable name (in line 112) is `kReturnIDPCredentialKey`. Since this isn't related to this PR and `IDP` is used in variable names elsewhere in the codebase, I'll leave this alone for consistency.